pkg/api/types.go

```diff
@@ -835,6 +838,10 @@ type PodStatus struct {
 	HostIP string `json:"hostIP,omitempty"`
 	PodIP  string `json:"podIP,omitempty"`
 
+	// Date and time at which the object was acknowledged by the Kubelet.
+	// This is before the Kubelet pulled the container image(s) for the pod.
```

Comment:
Ideally, we should separate the limit of pulling all images in a pod from how long a container in pod could run, but that is much more complicated today. Also I feel one bound limit to the end user is much easy to understand.